tez-api/src/main/java/org/apache/tez/dag/api/TezConfiguration.java

```diff
@@ -408,6 +409,18 @@ public TezConfiguration(boolean loadDefaults) {
       + "launch.env";
   public static final String TEZ_AM_LAUNCH_ENV_DEFAULT = "";
 
+  /**
+   * String value. Describes the timing of scheduling downstream vertex tasks
+   * when the vertex has a concurrent input edge.
```

Comment:
I have expand the comment a bit in an effort to make it easier to understand. 

Not using the event name directly is due to the subtle difference between event and scheduling actually being kicked off: for example, in the case of "SOURCE_TASK_STARTED", the scheduling is not triggered by any TASK_STARTED event, but instead it will only be triggered after enough TASK_STARTED events have been collected (for example, all the upstream tasks are running). For the lack of a better choice, I think defining a TriggerType is easier to expand in the long run too.